### ISSUE TITLE
Documented way just hangs during test

### ISSUE THREAD

**Issue by gmineebates** [codecov/example-objc#5]
On Xcode 8 environment, `-enableCodeCoverage=YES` works but the documented `GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES` does not.

**Comment by stevepeak** [comment]
@gmineebates Thank you for those suggestions. I belive this is better suited for https://github.com/codecov/example-swift right?

**Comment by gmineebates** [comment]
@stevepeak I didn't check out that repository, it looks like you guys do it with GUI in `Enable "Gather coverage data" in your test scheme:` which I wish I knew about. 😅
So to answer your question, the https://github.com/codecov/example-objc repository still instructs you use the `GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES` which didn't work with Xcode 8, Objective-C environment.

**Comment by stevepeak** [comment]
I see. Hum... Perhaps it would be best to add this as a caveat on the readme. Thoughts?

**Comment by gmineebates** [comment]
Yeah, I would add the same steps mentioned in https://github.com/codecov/example-swift
If it's through GUI, it'll most likely be future-compatible.

**Comment by RyanCopley** [comment]
It may be outdated. Feel free to open a PR with the correct documentation 👍
